contracts/xsc_connector/src/lib.rs

```diff
@@ -12,6 +12,8 @@ use utils::{hashes, Hash};
 
 const NO_DEPOSIT: Balance = 0;
 
+const MAX_DESTINATION_GAS: Gas = Gas(50_000_000_000_000);
```

Comment:
lets increase this to 200Tgas to allow for more complex method calls on the other side